### ISSUE TITLE
Plugin sometimes fails on VS start

### ISSUE THREAD

**Issue by wilhen01** [jzoss/Git-Source-Control-Provider#21]
![image](https://cloud.githubusercontent.com/assets/4677718/11954148/0a5cafc0-a89e-11e5-8853-15e235346a6b.png)

Seen on Visual Studio start intermittently, usually fixed by closing and reopening. Activity log has the following:

```<entry>
    <record>493</record>
    <time>2015/12/22 11:16:16.702</time>
    <type>Error</type>
    <source>VisualStudio</source>
    <description>Construction of frame content failed.&#x000D;&#x000A;Frame identifier: ST:0:0:{75edecf4-68d8-4b7b-92a9-5915461da6d9}&#x000D;&#x000A;Frame caption: Pending Changes&#x000D;&#x000A;Exception details:&#x000D;&#x000A;System.ArgumentNullException: Value cannot be null.&#x000D;&#x000A;Parameter name: key&#x000D;&#x000A;   at System.Collections.Concurrent.ConcurrentDictionary`2.TryGetValue(TKey key, TValue&amp; value)&#x000D;&#x000A;   at GitScc.RepositoryManager.GetTrackerForPath(String filename, Boolean createTracker)&#x000D;&#x000A;   at GitScc.PendingChangesToolWindow.OnToolWindowCreated() in C:\DevelExamples\Git-Source-Control-Provider\GitSccProvider\PendingChangesToolWindow.cs:line 66&#x000D;&#x000A;   at Microsoft.VisualStudio.Shell.Package.CreateToolWindow(Type toolWindowType, Int32 id, UInt32 flags)&#x000D;&#x000A;   at Microsoft.VisualStudio.Shell.Package.CreateToolWindow(Type toolWindowType, Int32 id, ProvideToolWindowAttribute tool)&#x000D;&#x000A;   at Microsoft.VisualStudio.Shell.Package.FindToolWindow(Type toolWindowType, Int32 id, Boolean create, ProvideToolWindowAttribute tool)&#x000D;&#x000A;   at Microsoft.VisualStudio.Shell.Package.CreateToolWindow(Guid&amp; toolWindowType, Int32 id)&#x000D;&#x000A;   at Microsoft.VisualStudio.Shell.Package.Microsoft.VisualStudio.Shell.Interop.IVsToolWindowFactory.CreateToolWindow(Guid&amp; toolWindowType, UInt32 id)&#x000D;&#x000A;   at Microsoft.VisualStudio.Platform.WindowManagement.WindowFrame.ConstructContent()</description>
  </entry>
```

I'm using v1.4.2

**Comment by jzoss** [comment]
I am pretty sure I already fixed this in the pre-1.4.3 release.. but I will double check.

**Comment by wilhen01** [comment]
Thanks @jzoss! Haven't had time to install the pre-1.4.3 build yet, and I'm on holiday from tomorrow until next year so won't happen til Jan now.

**Comment by jzoss** [comment]
Don't worry..I'm doing the same thing.. Just finishing up at work before my holiday.. Have a good one.. Talk you you next year!

**Comment by jzoss** [comment]
Fixed in 1.5
